### PR TITLE
Update oxford-metadata.ttl

### PR DESCRIPTION
**Pull request by MauriceHendrix**
For chaste code generation we needed a way to differentiate between membrane currents and their properties that are not actually currents themselves, so that we can create derived quantities in chaste units for just the currents without inadvertently converting their properties.

### CODE REVIEW

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl:152]: This will mean `IonicCurrent` doesn't show up as a category in its own right in the annotation tool. Is that reasonable @mirams @MichaelClerx ? All terms in this category should also appear in another category, and I wanted to reduce duplication in the tool.

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]: You probably want to adapt these comments in a similar way to `MembranePumpRelated`?

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]: This class has not been renamed; similarly for others. We should probably be consistent!

Comment (MauriceHendrix) [reply oxford-metadata.ttl]: like now?

File: oxford-metadata.ttl
Comment (MauriceHendrix) [reply oxford-metadata.ttl]: like now?

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]: You've not renamed this one, for instance. And other terms down to SRIonChannel. But possibly we don't need to since their existing names don't imply they're currents in the same way? What do you think @mirams ?

File: oxford-metadata.ttl
Comment (mirams) [reply oxford-metadata.ttl:152]: I think that's fine for now.

File: oxford-metadata.ttl
Comment (mirams) [reply oxford-metadata.ttl]: Yeah, for consistency I think these probably want to become 'Related' suffix, because they are labelling things that aren't actually the channel/current but properties of it. Same for MembranePotassiumChannel etc.

File: oxford-metadata.ttl
Comment (mirams) [inline oxford-metadata.ttl:364]: For the stimulus current do we also want `StimulusCurrent` (itself, this one) and `StimulusCurrentRelated`?

File: oxford-metadata.ttl
Comment (MauriceHendrix) [reply oxford-metadata.ttl]: ok @mirams @MichaelClerx @jonc125 Just ypdated it please have a look if this is what you meant

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]:
```suggestion
    rdfs:label "Exchangers" ;
```
Because I think the plural makes more sense as a label on the annotation tree.

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]:
```suggestion
    rdfs:label "Terms related to mixed ion channels" ;
```

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]:
```suggestion
    rdfs:label "Pumps"  ;
    rdfs:comment "Terms related to proteins in the sarcoplasmic reticulum membrane that use energy in ATP to pump ions against their electrochemical gradients." .
```
For consistency with the membrane channels

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl:219]: Let's also replace the '.' here with
```
;
rdfs:comment "Terms related to proteins in the sarcoplasmic reticulum membrane that use the energy of one ionic species moving down an electrochemical gradient, to move another species against its gradient." .
```

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]:
```suggestion
:SRIonChannelRelated
    a :SRCurrentRelated ;
    rdfs:label "Ion channels" ;
    rdfs:comment "Terms related to proteins in the sarcoplasmic reticulum membrane allowing certain ions to flow down their electrochemical gradients." .
```

File: oxford-metadata.ttl
Comment (jonc125) [inline oxford-metadata.ttl]:
```suggestion
:SRPumpRelated
```

**Review by jonc125 — APPROVED** [review]
Just some minor issues left I think before this can be merged (with updated RDF XML).

At some point @mirams should probably go through the whole file and review the labels & comments, but that doesn't need to hold this up!

**Comment by mirams** [comment]
Yep, Jon's changes look good, I'm happy

**Review by mirams — APPROVED** [review]

**Pull request merged**
